CHANGELOG.next.asciidoc

```diff
@@ -533,6 +533,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add support for named ports in autodiscover. {pull}19398[19398]
 - Add param `aws_partition` to support aws-cn, aws-us-gov regions. {issue}18850[18850] {pull}19423[19423]
 - The `elasticsearch/index` metricset now collects metrics for hidden indices as well. {issue}18639[18639] {pull}18703[18703]
+- Added to MySQL `performance` as lightweight metricset and `query` as parent lightweight metricset {pull}18955[18955]
```

Comment:
Users don't mind if they are lightweight or not :slightly_smiling_face: 
```suggestion
- Added `performance` and `query` metricsets to `mysql` module. {pull}18955[18955]
```

Reply:
Ok, changed.